src/Kentico.Xperience.Lucene/Models/IndexedItemModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Kentico.Xperience.Lucene.Models;
+
+public class IndexedItemModel
```

Comment:
This and the `IndexedContentItemModel` should include more of the fields available on the `WebPageEventArgsBase` or `ContentItemEventArgsBase`.

Like `ID`, `Name`, `IsSecured`

Reply:
Added